src/app/components/manuscript-thumbnails-viewer/manuscript-thumbnails-viewer.component.scss

```diff
@@ -0,0 +1,56 @@
+.thumbnails-view{
+    height: 100%;
+    background: #617d8a;
+    overflow: scroll;
+    position: relative;
+}
+.change-page-container {
+    padding: 10px;
+    background: #273238;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+}
+.page{
+    display: none;
+    padding: 0 20px;
+}
+.active {
+    display: block;
+}
+.item-content{
+    position: relative;
+}
+p.item-page-index {
+    position: absolute;
+    top: 70%;
+    width: calc(100% - 16px);
```

Comment:
use the mixin `calc` from `assets/scss/mixins.scss` file, in order to automatically add support for very old browser! After importing `mixins.scss` the file, you can use it as follow:
```
@include calc('width', '100% - 16px');
```